doc(release) Release notes Cloud 25.01 (#4118)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md

title: Centreon Cloud - notes de release
---

## March xxth, 2025

### Centreon IT Edition

**Centreon Monitoring Agent Beta**

> Beta of Centreon Monitoring Agent is now available for our Centreon Cloud customers.

Reviewer: ```suggestion
The Beta version of the Centreon Monitoring Agent is now available for our Centreon Cloud customers.
```

The Centreon Monitoring Agent (CMA) collects metrics and computes statuses on the servers it monitors, and sends them to Centreon. Centreon plugins as well as Nagios-compatible plugins can be used with this agent.

> More information about configuration are available in our [offical documentation](https://docs.centreon.com/pp/integrations/plugin-packs/getting-started/how-to-guides/cma/) and in a dedicated [TheWatch group](https://thewatch.centreon.com/groups/opentelemetry-agent-beta-program-61).

Reviewer: ```suggestion
More information about configuration is available in our [offical documentation](/pp/integrations/plugin-packs/getting-started/how-to-guides/cma/) and in a dedicated [TheWatch group](https://thewatch.centreon.com/groups/opentelemetry-agent-beta-program-61).
```

> **OpenTicket**

Reviewer: ```suggestion
**Open Ticket**
```

> Centreon Open Tickets, our module to create tickets to ITSM platform from Centreon is now available.

Reviewer: ```suggestion
Centreon Open Tickets, our module to create tickets into an ITSM platform directly from Centreon is now available.
```

> Once a provider is configured, the module allows for an operator to create tickets for hosts and services in a non-ok state using Centreon Dashboard resource table widget.

Reviewer: ```suggestion
Once a provider is configured, the module allows for an operator to create tickets for hosts and services in a non-OK status, using the Resource table widget in dashboards.
```

**Event Handler**

> Event handlers are optional system commands (scripts or executables) that are run whenever a resource state change occurs. 

Reviewer: ```suggestion
Event handlers are optional system commands (scripts or executables) that are run whenever a resource status change occurs. 
```

> Define your event handler command in Centreon interface and authorize them on your poller thanks to whitelists.

Reviewer: ```suggestion
Define your event handler command in the Centreon interface and authorize it on your poller using whitelists.
```

**Centreon Dashbaords**

> As usual, we continue to make improvment to our Centreon Dashbaord feature. 

Reviewer: ```suggestion
As usual, we continue to make improvements to our dashboard feature. 
```

> You can now mark dashboards as favorite for immediate access.

Reviewer: ```suggestion
You can now mark a dashboard as favorite for immediate access.
```

> It is also possible to expand widget to get a better visibility on specific data.

Reviewer: ```suggestion
It is also possible to expand widgets to get a better visibility on specific data.
```

> Home page has also been improved with dashboard thumbnails which allows to identify dashbaord easily.

Reviewer: ```suggestion
The home page has also been improved with dashboard thumbnails which allows to identify dashboards easily.
```

### Centreon Business Edition

**Centreon MAP**

> This version brings significant performance enhancement.

Reviewer: ```suggestion
This version brings significant performance enhancements.
```

> Other minor improvments have been made such as tooltip design and link to Resource Status, the ability to combine tile background color with weather icons to represent status of a resource, the possibility to open links in URL widget a new tab and the display of parent name in Gauge widget.

Reviewer: ```suggestion
Other minor improvements have been made: improvements to the design of tooltips, a link to the Resource Status page, the ability to combine tile background color with weather icons to represent the status of a resource, the possibility to open the links in URL widgets to a new tab and the display of the resource's parent name in Gauge widgets.
```
